packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -49,53 +48,125 @@ const WalletTransfer = observer(() => {
-                    mobile_list_min_height={account_list_height_with_offset}
-                    onSelectAccount={setFromAccount}
-                    placeholder={localize('Select a trading account or a Wallet')}
-                    portal_id={portal_id}
-                    transfer_accounts={transfer_accounts}
-                    // TODO: 'Demo USD Wallet' should be replaced to the current open wallet after connecting to API call
-                    wallet_name='Demo USD Wallet'
-                    value={from_account}
-                />
-            </div>
-            <div className='wallet-transfer__tile'>
-                <AmountInput
-                    currency={to_account?.currency || ''}
-                    decimal_places={to_account?.currency ? getDecimalPlaces(from_account?.currency) : 0}
-                    disabled={!to_account}
-                    initial_value={to_amount}
-                    label={localize('Amount you receive')}
-                />
-                <Divider />
-                <TransferAccountSelector
-                    is_mobile={is_mobile}
-                    label={localize('Transfer to')}
-                    mobile_list_min_height={account_list_height_with_offset}
-                    onSelectAccount={setToAccount}
-                    placeholder={!to_account ? localize('Select a trading account or a Wallet') : ''}
-                    portal_id={portal_id}
-                    transfer_accounts={to_account_list}
-                    transfer_hint={transfer_to_hint}
-                    // TODO: 'Demo USD Wallet' should be replaced to the current open wallet after connecting to API call
-                    wallet_name='Demo USD Wallet'
-                    value={to_account}
-                />
-            </div>
+            <Formik
+                initialValues={{
+                    to_amount: 0,
+                    from_amount: 0,
+                }}
+                onSubmit={() => {
+                    // console.log('submit');
+                }}
+                validateOnBlur={false}
+                // validate={values => {
+                //     console.log('values', values);
+                // }}
+            >
+                {({ setFieldValue, values }) => {
+                    const onSelectToAccount = React.useCallback(
+                        (account: TAccount) => {
+                            setToAccount(account);
+                            setFieldValue('to_amount', values.from_amount);
+                        },
+                        [setFieldValue, values.from_amount]
+                    );
+
+                    return (
+                        <Form noValidate>
+                            <div className='wallet-transfer__tiles-container'>
+                                <div className='wallet-transfer__tile'>
+                                    {/* <Field name='to_amount' validate={validateAmount}> */}
```

Comment:
@heorhi-deriv is this commented code required?

Reply:
@aum-deriv will remove it